### PR TITLE
Update readme to call out change from Managed VMs to App Engine Flexible Environment

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

**Comment by jonparrott** [comment]
Hey, @JustinBeckwith - fix your CLA. :P

File: README.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
-Google App Engine Managed VMs Python Compat Runtime
+Google App Engine Flexible Environment Python Compat Runtime
 ===================================================
 
-Warning
--------
-The instructions here are for a multi-process version of the Python 2.7 runtime
-**currently in alpha**. To use the stable version of the runtime, follow the
```
Comment (jonparrott) [inline README.md]: @JustinBeckwith note, this runtime itself is still in alpha. Please note that in your new readme.

Comment (JustinBeckwith) [inline README.md]: Wait - so the previous python27 runtime was beta, and we changed the default in gcloud to drop a dockerfile for an image in alpha?

**Comment by JustinBeckwith** [comment]
Well that's weird - usually it picks up that I work here and don't need to sign a cla :)  Do I need to do anything?

File: README.md (hunk 1 above)
Comment (bryanmau1) [inline README.md]: Yes, this runtime is in alpha.

Comment (jcollins-g) [inline README.md]: Until end-to-end tests are more extensive and cover at least essential AppEngine API functionality, I'm not comfortable calling this runtime Beta.

**Comment by JustinBeckwith** [comment]
Is this better?

**Comment by jonparrott** [comment]
Looks good, merging.

**Pull request merged**